WcaOnRails/app/models/round.rb

```diff
@@ -66,13 +71,13 @@ def name
 
   def time_limit_to_s
     time_str = SolveTime.new(competition_event.event_id, :best, time_limit.centiseconds).clock_format
-    case self.time_limit.cumulative_round_ids.length
+    case self.time_limit.cumulativeRoundIds.length
```

Comment:
I guess that you meant camelCase when saying *snake_case* and the other too, right? ^^